dmutils/email/tokens.py

```diff
@@ -100,7 +102,11 @@ def decode_password_reset_token(token, data_api_client):
         decoded, token_timestamp = decode_token(
             token,
             current_app.config["SHARED_EMAIL_KEY"],
-            current_app.config["RESET_PASSWORD_SALT"],
+            (
+                current_app.config.get('RESET_PASSWORD_TOKEN_NS')
```

Comment:
I might be missing something here - what does `_NS` stand for? 'Not salt'?

Reply:
Namespace.